data/categories.txt

```diff
@@ -4900,34 +4913,96 @@ sk:4^Kúpalisko|3bazén|aquapark|4kúpele
 sw:^Bwawa la kuogelea
 
 sport-equestrian
-en:Equestrian Sports
-ru:Конный спорт
-de:Pferdesport
-es:Deportes ecuestres
-fr:Sport équestre
-hu:Lovassportok
-nl:Paardensport
+en:^Equestrianism
```

Comment:
Please add "riding" as a synonym. After all, these are supposed to help our search engine.